Add-IN/ACDC4Robot/commands/ACDC4Robot/acdc4robot.py

```diff
@@ -177,29 +178,15 @@ def run():
             if simulator  == "None":
                 ui.messageBox("Simulation environment is None.\n" +
                               "Please select a simulation environment.", msg_box_title)
-            elif simulator == "Gazebo":
-                # write to .urdf file
-                write.write_urdf(link_list, joint_list, save_folder, robot_name)
-                # export mesh files
-                export_stl(design, save_folder, link_list)
-                ui.messageBox("Finished exporting URDF for Gazebo.", msg_box_title)
-                
-            elif simulator == "PyBullet":
+            elif simulator in ["Gazebo", "PyBullet", "MuJoCo"]:
```

Comment:
Since the code in each condition was nearly identical, I combined them into a single block.